# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/doc/Debugger/Debugger.Frame.md
# Commit: fa46d8caefde
# Full Hash: fa46d8caefdeea83b7a41ba3259114c13dd3b145
# Author: Logan Smyth <loganfsmyth@gmail.com>
# Date: 2019-12-09 09:50:39
# Regressor Bug: 1600204
# File Overlap Count: 1
# Description:
#   Bug 1600204 - Part 6: Support .older on suspended generator frames. r=jimb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55236
# ==============================================================================

diff -r ea1bb2f0fd39 -r fa46d8caefde js/src/doc/Debugger/Debugger.Frame.md
--- a/js/src/doc/Debugger/Debugger.Frame.md	Sun Dec 08 23:53:27 2019 +0000
+++ b/js/src/doc/Debugger/Debugger.Frame.md	Sun Dec 08 23:54:15 2019 +0000
@@ -205,9 +205,10 @@
 
 ### `older`
 The next-older visible frame, in which control will resume when this
-frame completes. If there is no older frame, this is `null`.
+frame completes. If there is no older frame, this is `null`. If this frame is
+a suspended generator or async call, this will also be `null`.
 
-Accessing this property will throw if `.onStack == false`.
+Accessing this property will throw if `.terminated == true`.
 
 ### `onStack`
 True if the frame this `Debugger.Frame` instance refers to is still on